binders/rabbit-binder/spring-cloud-stream-binder-rabbit/src/test/java/org/springframework/cloud/stream/binder/rabbit/RabbitBinderCleanerTests.java

```diff
@@ -184,34 +186,8 @@ private void waitForConsumerStateNot(String queueName, long state) throws Interr
 		assertThat(cleanedExchanges).hasSize(6);
 	}
 
-	public static class AmqpQueue {
-
-		private boolean autoDelete;
-
-		private boolean durable;
-
-		public AmqpQueue(boolean autoDelete, boolean durable) {
-			this.autoDelete = autoDelete;
-			this.durable = durable;
-		}
-
-		@JsonProperty("auto_delete")
-		protected boolean isAutoDelete() {
-			return autoDelete;
-		}
-
-		protected void setAutoDelete(boolean autoDelete) {
-			this.autoDelete = autoDelete;
-		}
-
-		protected boolean isDurable() {
-			return durable;
-		}
-
-		protected void setDurable(boolean durable) {
-			this.durable = durable;
-		}
-
+	private static Map<String, List<String>> doClean(RabbitBindingCleaner cleaner, String entity, boolean isJob) {
+		return cleaner.clean(RABBITMQ.getHttpUrl() + "/api", "guest", "guest", "/", BINDER_PREFIX, entity, isJob);
```

Comment:
Had to use `clean` method that accepted "all the things" as we needed to configure the port - the cleaner default is hardcoded to `15672`.